consensus-types/types/payload.go

```diff
@@ -568,7 +568,7 @@ func (p *ExecutionPayload) ToHeader() (
 	txsRoot := p.GetTransactions().HashTreeRoot()
 
 	switch p.Version() {
-	case version.Deneb, version.DenebPlus:
```

Comment:
note: version.DenebPlus got never activated so this is backward compatible change